### PR TITLE
Support for /user/info endpoint

### PR DESCRIPTION
**Pull request by hvge**
This PR adds support for `/user/info` endpoint and improves error reported when the request encryption is required in the wrong activation state.

### CODE REVIEW

File: proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/ecies/EciesEncryptorFactory.java [hunk 1]
```diff
@@ -77,23 +77,26 @@ public EciesEncryptorFactory(@NonNull Session session, @Nullable byte[] possessi
      * @throws PowerAuthErrorException if factory doesn't have {@link #mPossessionUnlockKey} but is required,
      *                                 or if low level encryptor creation fails
      */
-    private @NonNull EciesEncryptor getEncryptor(@NonNull @EciesEncryptorScope int scope, @Nullable String sharedInfo1, boolean addMetaData) throws PowerAuthErrorException {
+    private @NonNull EciesEncryptor getEncryptor(@EciesEncryptorScope int scope, @Nullable String sharedInfo1, boolean addMetaData) throws PowerAuthErrorException {
```
Comment (kober32) [inline proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/ecies/EciesEncryptorFactory.java:80]: Do I understand correctly, that the `int` cannot be null so the attribute was unnecessary?

**Review by kober32 — CHANGES_REQUESTED** [review]
Looks OK, but there is no documentation?

File: proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/ecies/EciesEncryptorFactory.java (hunk 1 above)
Comment (hvge) [reply proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/ecies/EciesEncryptorFactory.java:80]: Yep, there was a warning about this.

**Comment by hvge** [comment]
The documentation will be addressed in the separate ticket, see #512

**Review by kober32 — APPROVED** [review]
OK then

**Pull request merged**